doc/man3/EVP_MD_fetch.pod

```diff
@@ -11,60 +11,88 @@ EVP_MD_fetch
 
  EVP_MD *EVP_MD_fetch(OPENSSL_CTX *ctx, const char *algorithm,
                       const char *properties);
+ EVP_CIPHER *EVP_CIPHER_fetch(OPENSSL_CTX *ctx, const char *algorithm,
+                              const char *properties);
 
 =head1 DESCRIPTION
 
-The B<EVP_MD> object is used for representing a digest method implementation.
+Cryptographic algorithms are represented by different OpenSSL objects depending
```

Comment:
What about something like
'The following Cryptographic algorithm types are supported.' And then itemize the types.

Reply:
Done. I added EVP_MAC and EVP_KDF as well, even though we don't yet have "fetch" implementations for them.

Reply:
Please remind me to adapt in #8877